src/components/CardDropdown/index.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+
+import { Button, Dropdown, Menu, message, Typography } from 'antd';
+import { DownOutlined, EllipsisOutlined } from '@ant-design/icons';
+
+import CIcon from 'assets/img/c_icon.svg';
+import JavaScriptIcon from 'assets/img/javascript_icon.svg';
+import PythonIcon from 'assets/img/python_icon.svg';
+
+import './styles.scss';
+
+const { Link } = Typography;
+
+function handleMenuClick() {
+  message.info('Click on menu item.');
+}
+
+const menu = (
+  <Menu className="menu-icon" onClick={handleMenuClick}>
+    <Menu.Item key="1" icon={<img alt="Python" src={PythonIcon} />}>
```

Comment:
it would be better if we put this in a config file then loop its contents here. 

For example:
```js
const languagesMenuConfig = {  
  PYTHON: {
     icon: PythonIcon
     label: "Python"
  },
  ...etc
}
```

and there should also be another constant file for all languages

```
// languages.json

{
   "PYTHON": 'python',
   "JAVASCRIPT": 'js'
}
```

take this as an example to show the intent. 

if I remember it correctly I think `react-ace` has this also.